OCPBUGS#7152: Adds more detail for the agent create command (#57042)
openshift/openshift-docs: modules/web-console-overview.adoc

Use those details to log in and access the web console.

For existing clusters that you did not install, you can use `oc whoami --show-console` to see the web console URL.

[IMPORTANT]
====

> The `dir` parameter specifies the assets directory, which stores the manifest files, the ISO image, and ** auth ** directory. The ** auth ** directory stores the ** kubeadmin-password ** and ** kubeconfig ** files. As a `kubeadmin` user, you can use `kubeconfig` to access the cluster with the following setting: `export KUBECONFIG=<install_directory>/auth/kubeconfig`. Note that the `kubeconfig` is specific to the generated ISO image, so if the `kubeconfig` is set and the `oc` command fails, it is possible that the system did not boot with the generated ISO image. You can use the contents of ** kubeadmin-password ** file to log into the console during the bootstrap as the `core` user to perform debugging.

Reviewer: ```suggestion
The `dir` parameter specifies the `assets` directory, which stores the manifest files, the ISO image, and the `auth` directory. The `auth` directory stores the `kubeadmin-password` and `kubeconfig` files. As a `kubeadmin` user, you can use the `kubeconfig` file to access the cluster with the following setting: `export KUBECONFIG=<install_directory>/auth/kubeconfig`. The `kubeconfig` is specific to the generated ISO image, so if the `kubeconfig` is set and the `oc` command fails, it is possible that the system did not boot with the generated ISO image. To perform debugging, during the bootstrap process, you can log in to the console as the `core` user by using the contents of the `kubeadmin-password` file.
```

- Per the IBM Style guide, directory and file names should be in monospace font. (See https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#highlighting-by-element).
- I'm thinking that "...you can use `kubeconfig` to access..." should be "...you can use the `kubeconfig` file to access...", but feel free to ignore me if that's not technically accurate.

Author: Addressed

> ====

Author: @bfournie Can you please review?

Reviewer 2: This looks fine but should we create a separate bug to add additional troubleshooting info as suggested by Zane in https://github.com/openshift/openshift-docs/pull/55668#discussion_r1105243059

Author: Yes. We can create a separate bug to address that. For now, I will get this PR merged.
